dashboards/system_tables/sql/job_concurrency_comparison.sql

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * It is assumed that the following query will be run in the administration
+ * project where the reservations were created. If this is not the case,
+ * prepend the project id to the table name as follows:
+ * `{project_id}`.`region-{region_name}`.INFORMATION_SCHEMA.{table}
+ */
+
+/*
+ * Job Comparison Report: Returns information about jobs to compare performance
+ * and troubleshoot.
+ */
+WITH
+  job_info AS (
+  SELECT
+    creation_time,
+    end_time,
+    project_id,
+    reservation_id
+  FROM
+    `region-{region_name}`.INFORMATION_SCHEMA.JOBS_BY_ORGANIZATION
+  WHERE
+    job_id = @job_param ),
+  reservation_capacity AS (
+  SELECT
+    slot_capacity,
+    creation_time,
+    end_time,
+    job_info.reservation_id
+  FROM (
+    SELECT
+      slot_capacity,
+      CONCAT(project_id, ":", "US", ".", reservation_name) AS reservation_id,
```

Comment:
Do we have instructions to change this region in the case of differences?

Reply:
Not sure why I uploaded this one - should just be slow and fast